utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -1200,6 +1220,9 @@ def create_argument_parser():
            help='skip building cmark')
     option('--skip-build-llvm', toggle_false('build_llvm'),
            help='skip building llvm')
+    option('--build-llvm', toggle_true('_build_llvm'),
+           default=True,
+           help='build llvm and clang')
```

Comment:
This is going to be confusing, though you are simply reproducing the current confusing status quo, ie there are three possible states prior to this pull:

1. build the full LLVM
2. `--skip-build-llvm` builds only a few minimal utilities like `llvm-tblgen`.
3. `--build-llvm=0` genuinely builds nothing from LLVM.

Can we take this opportunity to clean this up or document this, ie add some doc here in the flags on what `--build-llvm=False` does?

Reply:
We can‘t change it, because we‘d break builds that depend on it, but proper documentation is a good point.